Program.cs

```diff
@@ -248,11 +248,13 @@ public class DotNetWebServer
         protected Socket clientSocket;
         private static DotNetWebServer _instance = null;
         protected int id;
+        protected int MaxThread;                                                       //Max Thread pool select from config.json
+        protected String ThreadModel;                                                  //Thread model select from config.json
 
         private DotNetWebServer(Program parent)
         {
             _parent = parent;
-            id = 0;
+            id = 0;                  
```

Comment:
Be aware of any commit change that doesn't do anything. You should review your changes before commiting.